schema_source/jetstream/api/v1/definitions.json

```diff
@@ -833,7 +837,8 @@
         "inactive_threshold": {
           "description": "Duration that instructs the server to cleanup ephemeral consumers that are inactive for that long",
           "$ref": "#/definitions/golang_duration_nanos",
-          "default": 0
+          "default": 0,
+          "minimum": 0
```

Comment:
since golang_duration_nanos defines a minimum we should be able to drop minimum here.